src/options.c

```diff
@@ -1484,7 +1484,7 @@ read_repo_config_option(char *name, size_t namelen, char *value, size_t valuelen
 	else if (!strcmp(name, "diff.noprefix"))
 		parse_bool(&opt_diff_noprefix, value);
 
-	else if (!strcmp(name, "status.showUntrackedFiles"))
+	else if (!strcmp(name, "status.showuntrackedfiles"))
```

Comment:
nice fix, thanks!
If possible, could you update the commit subject (the first line in the commit message) to mention `status.showUntrackedFiles` like the PR subject does? The commit subject will appear in the release notes (via git-shortlog).

Reply:
Done!

Reply:
Thanks.